matview.c

```diff
@@ -1050,9 +1051,9 @@ IVM_immediate_maintenance(PG_FUNCTION_ARGS)
 			char   *count_colname = NULL;
 
 			/* check if the modified table is in EXISTS clause. */
-			for (i = 0; i < list_length(rte_path); i++)
+			for (int j = 0; j < list_length(rte_path); j++)
```

Comment:
Is there a reason why you changed this from 'i' to 'j'?

Reply:
I intended to add the counter variable for suppress compiler warning due to shadow declaration, but it was unnecessary, so I removed it.

Reply:
OK, thanks